tools/test.py

```diff
@@ -1479,6 +1500,17 @@ def Main():
   def DoSkip(case):
     return SKIP in case.outcomes or SLOW in case.outcomes
   cases_to_run = [ c for c in all_cases if not DoSkip(c) ]
+  if options.run != None:
```

Comment:
The proper way to check `None` is `if options.run is None`. Please check [this](http://stackoverflow.com/a/23086405/1903116)